src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -21,9 +24,26 @@ public void getTrelloBoards() {
 
         List<TrelloBoardDto> trelloBoards = trelloClient.getTrelloBoards();
 
-        trelloBoards.stream()
-                .filter(c -> c.getName().contains("Kodilla"))
-                .forEach(trelloBoardDto -> System.out.println(trelloBoardDto.getId() + " " + trelloBoardDto.getName()));
+        trelloBoards.forEach(trelloBoardDto -> {
 
+            System.out.println(trelloBoardDto.getName() + " - " + trelloBoardDto.getId());
```

Comment:
a może by tak podejść do tego trochę inaczej i zamiast `System.out.println`, napisać test integracyjny, który będzie wołał endpoint `/getTrelloBoards` <- wtedy też ten endpoint musiałby coś zwracać, zapewne listę board'ów, teraz zwraca `void` - to też trochę wprowadza w błąd, niby metoda **get**TrelloBoards, a nic nie zwraca.